src/Goerli-DssSpell.t.sol

```diff
@@ -16,6 +16,7 @@
 
 pragma solidity 0.6.12;
 
+import "dss-exec-lib/DssExecLib.sol";
```

Comment:
why did you add this? 
`Changelog` is declared in [Goerli-DssSpell.t.base.sol](https://github.com/makerdao/spells-goerli/blob/master/src/Goerli-DssSpell.t.base.sol#L64) and it uses the `chainlog` address specified in `address_goerli.sol`

Reply:
I wanted to remove the hard-coded chainlog address from the helper spells.
In the end I removed this import to use the `Addresses` contract instead.